app/components/person-search.hbs

```diff
@@ -96,7 +96,9 @@
     </AuDataTable>
   </div>
 {{else}}
-  <p class="au-u-margin au-u-margin-top-huge">
-    Zoek een reeds gekende persoon in de databank van het loket op naam.
-  </p>
+  <div class="au-o-box">
+    <AuAlert @alertIcon="info-circle" @alertTitle="Zoek een reeds gekende persoon in de databank van het loket op naam." @alertskin="info" @alertsize="small" class="au-u-margin-bottom-none">
```

Comment:
I'm not sure if this needed to be changed to an alert. The JIRA issue makes it sound like the text should have been appended to the existing message.

Reply:
It's more consistent to make this in to an alert. The message contains similar information as the other alerts so makes sense to design it the same way.

Reply:
The message says that you can search people by name. The not being able to add a person is just extra info, so I don't think it's the same use-case, but fair enough.